### ISSUE TITLE
Inputs randomly ignored if controller is connected to computer

### ISSUE THREAD

**Issue by nikso14** [godotengine/godot#62598]
### Godot version

v3.4.stable.official [206ba70f4]

### System information

OS: Windows 10, cpu: i5-7600k, gpu: nvidia gtx1060 6GB, rendering: GLES3

### Issue description

Input gets randomly ignored if controller is plugged in and button will require being released and pressed again to work correctly until it occurs again. In case project was not started with controller plugged in the issue will only appear after pressing buttons on controller. When using only controller the issue will be always present.

### Steps to reproduce

Was able to reproduce it in platformer example by making a long straight surface and moving character around.

**Comment by Calinou** [comment]
Related to https://github.com/godotengine/godot/issues/36396. Can you reproduce this after disabling **Vsync** in the Project Settings?

**Comment by nikso14** [comment]
yes it still happens both in example and my project(completely stops in my project but that is more likely due to my coding so I doubt it is related to this issue besides the momentary skip that appears in platfromer example where it seems to lose input for just a few frames)

**Comment by nikso14** [comment]
update on the issue, tried to use joystick on controller and it works correctly, only button type inputs seem to cause the issue (keyboard and d-pad)

**Comment by Calinou** [comment]
@nikso14 Please upload a [minimal reproduction project](https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#provide-a-simple-example-project) to make this easier to troubleshoot.

**Comment by nikso14** [comment]
[2d plat kinematic.zip](https://github.com/godotengine/godot/files/9788745/2d.plat.kinematic.zip)
bit different behavior than in my project (goes directly into idle state in my project) but it still skips inputs with controller plugged in and works fine when not

**Comment by nikso14** [comment]
Found a band-aid type of solution, unbinding joystick from action fixes the issue, was probably causing minor inputs and even if those inputs weren't past the deadzone it seems that godot was picking them up.

**Comment by Nintorch** [comment]
Hello! May I ask if the issue is still reproducible in Godot 4.5?
